CHANGELOG.md

```diff
@@ -1,281 +1,101 @@
-## [Sidero 0.5.0](https://github.com/talos-systems/sidero/releases/tag/v0.5.0) (2022-02-14)
```

Comment:
CHANGLOG seems to be messed up? 0.5.0 entry removed (?)

Reply:
oh... yeah